zaza/charm_tests/neutron/setup.py

```diff
@@ -72,6 +74,10 @@ def basic_overcloud_network():
     network_config.update(DEFAULT_UNDERCLOUD_NETWORK_CONFIG)
     # Environment specific settings
     network_config.update(generic_utils.get_undercloud_env_vars())
+    # Deployed model settings
+    if (model.get_application_config('neutron-api')
+       .get('enable-dvr').get('value')):
```

Comment:
It's merged but the indentation looks weird on this.

Reply:
I tried this with one more space:

```python3
if (model.get_application_config('neutron-api')
    .get('enable-dvr').get('value')):
```

and get:

```
zaza/charm_tests/neutron/setup.py:79:9: E129 visually indented line with same indent as next logical line
```

An alternate indentation that pep8 is happy with is:

```python3
if (model.get_application_config('neutron-api')
        .get('enable-dvr').get('value')):
```

I'm happy to propose an update to the second (deeper) indentation if you'd prefer.

Reply:
Resolving in #181